CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+# Copyright 2022
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+cmake_minimum_required(VERSION 3.1)
+project(secp256k1 VERSION 0.1.0 LANGUAGES C)
+
+set(CMAKE_C_STANDARD 90)
```

Comment:
Might be worth adding a comment that in cmake, 89/90 are the same option.

Reply:
I don't think that's necessary, they're also the same outside of cmake. "C89" and "C90" are just synonyms.